### PR TITLE
English changes by James

### PR DESCRIPTION
**Pull request by jamespjh**

### CODE REVIEW

**Comment by jamespjh** [comment]
Good work.

I made some changes for English readability.

@sinanshi review my changes.

File: report.md [hunk 1]
```diff
@@ -4,123 +4,157 @@
...
+either modified or removed according to these criteria, which are presented in a 
+``business readable'' file format allowing researchers to design and verify the criteria
+for record inclusion. An auto-generated data quality report is produced 
+to report on the rate of ``missingness'' of key fields, and on data sanity by site and unit. 
+
+By the end of the pipeline
+researchers obtain an R table which is queryable and
+validated. 
+
+We also have demonstrated the ability of programmatically
```
Comment (dpshelio) [inline report.md:35]: shouldn't be: `we have also demonstrated...` ?

File: report.md [hunk 2]
```diff
@@ -4,123 +4,157 @@
...
+about this process is in preparation.
+
+We believe robust, understandable and accessible software is
+the first step of solid computationally-based research. 
+Therefore, in addition to the functionalities described above, 
+we have also focused on the usability and sustainability of
+the software produced. 
+
+All of our development is constantly updated in the
+GitHub platform which is accessible to all the collaboration.
```
Comment (dpshelio) [inline report.md:48]: `accessible to all the collaborators`? or `to the whole collaboration`? I am not so sure of any of the options...

File: report.md [hunk 3]
```diff
@@ -4,123 +4,157 @@
...
+Therefore, in addition to the functionalities described above, 
+we have also focused on the usability and sustainability of
+the software produced. 
+
+All of our development is constantly updated in the
+GitHub platform which is accessible to all the collaboration.  
+The software is simple to install and configure.
+The code is self-explanatory and well
+documented empowering both current users and future
+developers. Full automated tests of code correctness
```
Comment (dpshelio) [inline report.md:52]: we don't care about future users :-P

File: report.md [hunk 4]
```diff
@@ -4,123 +4,157 @@
...
 Excel users or to be used with any other analysis tool or programming language.
 
-It is worth mentioning that a `C++` equivalent `XML` parser, which has a much better
-performance comparing to the currently used in R, has been developed but not deployed yet due
-to the time constraints of the project. The incorporation of the `C++` parser into the
-pipeline will boost the speed in which all the files are converted.
+We have on going work to develop an equivalent parser in `C++`. The prototype has a much better
+performance comparing to the `R` version, but has not yet been deployed into the current pipeline. 
+The incorporation of the `C++` parser into the
+pipeline will reduce from Xs to Ys the time to parse all the files.
```
Comment (dpshelio) [inline report.md:97]: Probably we need to get these numbers, right? @sinanshi let me know how I can get these numbers.

**Comment by dpshelio** [comment]
:+1:

**Pull request merged**